community/examples/xpk-gke-a3-megagpu.yaml

```diff
@@ -33,6 +33,7 @@ deployment_groups:
     source: modules/network/vpc
     settings:
       subnetwork_name: xpk-gke-a3-megagpu-subnet
+      mtu: 8244
```

Comment:
Curious, any reason to set the MTU to 8244?

Reply:
this is from the best practices from GKE from https://cloud.google.com/kubernetes-engine/docs/how-to/gpu-bandwidth-gpudirect-tcpx#create-vpcs-subnets